react/src/component/login/login.tsx

```diff
@@ -236,11 +236,13 @@ export const LoginExtraInfoForm = (props: { onComplete: ({displayName}) => void
                 <Form.Label column sm="3">
                     Your full name
                 </Form.Label>
-                <Col sm="9">
+                <Col sm="7">
                     <Form.Control value={displayName} onChange={(e) => updateDisplayName(e.target.value)}/>
                 </Col>
+                <Col sm="2">
+                    <Button variant="primary" onClick={() => onComplete({ displayName })} disabled={! displayName}>Continue</Button>
```

Comment:
displayName is valid should be a function. And OK button should only be clickable when all form fields are valid.